packages/vue/src/components/atoms/SfLink/SfLink.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+  <component
+    :is="linkComponentTag"
+    v-focus
+    :tab-index="disabled ? -1 : 0"
```

Comment:
@martaradziszewska IMO when the link is "disabled" then it is not a link but <span/> what did you think?